packages/base/SkillCard/card-editing.json

```diff
@@ -2,7 +2,7 @@
   "data": {
     "type": "card",
     "attributes": {
-      "instructions": "- If the user wants the data they see edited, AND the patchCard function is available, you MUST use the \"patchCard\" function to make the change.\n- If the user wants the data they see edited, AND the patchCard function is NOT available, you MUST ask the user to open the card and share it with you.\n- If you do not call patchCard, the user will not see the change.\n- You can ONLY modify cards shared with you. If there is no patchCard function or tool, then the user hasn't given you access.\n- NEVER tell the user to use patchCard; you should always do it for them.\n- If the user wants to search for a card instance, AND the \"searchCard\" function is available, you MUST use the \"searchCard\" function to find the card instance.\nOnly recommend one searchCard function at a time.\nIf the user wants to edit a field of a card, you can optionally use \"searchCard\" to help find a card instance that is compatible with the field being edited before using \"patchCard\" to make the change of the field.\n You MUST confirm with the user the correct choice of card instance that he intends to use based upon the results of the search.",
+      "instructions": "- If the user wants the card data they see edited, AND the patchCard function is available, you MUST use the \"patchCard\" function to make the change.\n- If the user wants the data they see edited, AND the patchCard function is NOT available, you MUST ask the user to open the card and share it with you.\n- If you do not call patchCard, the user will not see the change.\n- You can ONLY modify cards shared with you. If there is no patchCard function or tool, then the user hasn't given you access.\n- NEVER tell the user to use patchCard; you should always do it for them.\n- If the user wants to search for a card instance, AND the \"searchCard\" function is available, you MUST use the \"searchCard\" function to find the card instance.\nOnly recommend one searchCard function at a time.\nIf the user wants to edit a field of a card, you can optionally use \"searchCard\" to help find a card instance that is compatible with the field being edited before using \"patchCard\" to make the change of the field.\n You MUST confirm with the user the correct choice of card instance that he intends to use based upon the results of the search.\n- Do NOT use patchCard function when user is asking for template changes, visual modifications, style updates, or card behavior changes. patchCard should ONLY be used when explicitly changing card data values bound to the card definition in gts files, or data relationships, and nothing else.",
```

Comment:
Added: `- Do NOT use patchCard function when user is asking for template changes, visual modifications, style updates, or card behavior changes. patchCard should ONLY be used when explicitly changing card data values bound to the card definition in gts files, or data relationships, and nothing else."`